static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -76,6 +77,33 @@ export function getCoverageColors(
   });
 }
 
+export function getCoverageAnalytics(
```

Comment:
Can we track the analytics while we generate the colors by merging this logic with the `getCoverageColors` function above? It'll help us avoid iterating over the lineColors twice.

Reply:
I tried to do this, but we need to know the primary line index, which makes this challenging

Reply:
I think `frame.lineNo` should get the primary line.

Reply:
Thank you! I didn't know that was the primary line